CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 3.2.0
+
+### Added
+
+- Adds the unsigned version of `RATE` and `DELTA`, `URATE` and `UDELTA`, which 
```

Comment:
shouldn't be `positive` instead of `unsigned` ?
I would expect an unsigned to return the elapsed diff with non signed value, not to assert that diff is positive

Reply:
I changed them to `PRATE` and `PDELTA`, I tried `POSTIVE_RATE` and `POSITIVE_DELTA` but the linter complained that it was not CamelCase, WDYT?

Reply:
Linter is right consts should be camel-case unless they are borrowed from posix, but as we already have them in caps PDELTA lgtm